packages/modules/error-handling-react/tsconfig.json

```diff
@@ -10,7 +10,7 @@
       "dom.iterable",
       "scripthost"
     ],
-    "module": "commonjs",
+    "module": "es2020",
```

Comment:
let's undo this. We want to reset this package's config back to what it was previously (targeting 2017 i think? and commonjs)